solutions/swb-app/src/costCenterRoutes.ts

```diff
@@ -33,7 +34,16 @@ export function setUpCostCenterRoutes(
         CreateCostCenterRequestParser,
         req.body
       );
-      res.status(201).send(await costCenterService.create(validatedRequest));
+
+      try {
```

Comment:
nit: should catch all errors, check `isBoomError` and finally throw a `Boom.badImplementation` error if it is not a known error type